### DISCUSSION TITLE
Bug Noclip

### DISCUSSION THREAD

**Discussion by LeoPiedraYT** [maxnut/GDmenu#194] [Q&A]
In the platform mode El Noclip is bugged and the Noclip Accuracy comes out of 100% higher numbers and then when I enter a classic mode level, it begins to die in Noclip happens in versions 2.1 for 2.2 and 2.1 for 2.201
(https://github.com/maxnut/GDMegaOverlay/assets/97818448/55089217-cd0d-4a8f-9904-362e768dff19)

**Comment by MysticAx0lotl** [comment]
I legitamately do not understand any of this
